src/hooks/useNavigationEffect.js

```diff
@@ -0,0 +1,52 @@
+import { useEffect } from "react";
+
+import { MOUSE_EVENTS, KEYS, NON_NAVIGATION_KEYS } from "src/constants";
+
+export const useNavigationEffect = () => {
+  useEffect(() => {
+    if (typeof window !== "undefined" && typeof document !== "undefined") {
+      const eventListener = event =>
+        Object.values(KEYS).includes(event.key) &&
+        !NON_NAVIGATION_KEYS.includes(event.key) &&
```

Comment:
I think any time a key is pressed we should disable the mouse, no matter which one it is.

The only exception I would keep is when the key is `CMD`, `ALT, `windows`, `shift`, etc. In other words, modifier keys.

Is there an easy way to just check for modifier and let the rest pass the condition?

Reply:
Wouldn't be strange to let any key disable the mouse?

I mean, in a UX perspective, imagine pressing some number or the key `p` and the mouse disables, it would look like a bug, that's why I limited to only the mapped keys, so it feels like these keys, if you press, is gonna make an effect on the page.

Reply:
I don't think it would be a problem, because the events start again as soon as you move the cursor.

But I get where you come from. I can get behind that, and we'll leave it as is. Thank you for this discussion!